src/main/java/controllers/service/ArticleService.java

```diff
@@ -0,0 +1,32 @@
+package controllers.service;
```

Comment:
Service package not in the model package?

Reply:
I think it's in a coltrollers, no?